### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by gdbelvin**
This is a temporary hack to get the integration tests running again while #324 is being worked on.

Create databases before using them
- Creates connection test tree 0 on db reset
- Creates trees for integration tests

### CODE REVIEW

File: scripts/resetdb.sh [hunk 1]
```diff
@@ -8,5 +8,8 @@ then
     mysql -u root "$@" -e 'CREATE DATABASE test;'
     mysql -u root "$@" -e "GRANT ALL ON test.* TO 'test'@'localhost' IDENTIFIED BY 'zaphod';"
     mysql -u root "$@" -D test < storage/mysql/storage.sql
+    #TODO(codeinglama): remove when db connection check doesn't ping log 0
```
Comment (daviddrysdale) [inline scripts/resetdb.sh]: Spelt @codingllama

File: scripts/resetdb.sh [hunk 2]
```diff
@@ -8,5 +8,8 @@ then
...
+    mysql -u root "$@" -D test -e "INSERT INTO Trees VALUES (0,1,'LOG','SHA256','SHA256',false)" 
+    mysql -u root "$@" -D test -e "INSERT INTO TreeControl VALUES (0,false,false,false,1,1)"
```
Comment (daviddrysdale) [inline scripts/resetdb.sh]: Could this be done by just invoking `createlog.sh 0` ?

**Comment by daviddrysdale** [comment]
Would you expect Travis to work yet?

**Comment by gdbelvin** [comment]
This resolves the row missing errors. Travis may still fail due to too many DB connections at once.

**Comment by gdbelvin** [comment]
Rebased on @codingllama 's #326 to include the DB connections fix so Travis will pass.

**Comment by daviddrysdale** [comment]
Pull #328 seemed to fix things so has gone in....

**Pull request closed without merging**